### DISCUSSION TITLE
General question: SUEWS simulations in dusty days

### DISCUSSION THREAD

**Discussion by khorsans** [UMEP-dev/UMEP#430] [General]
Hi all,
Can the SUEWS scheme be used on dusty days as well? My goal is to study the changes in the amount of heat and radiation fluxes of a city on dusty days using the SUEWS scheme.

**Comment by suegrimmond** [comment]
Yes - if you have the forcing meteorology for those conditions.
The surface resistances have not been developed or tested under those conditions specifically. But depending on where you are thinking of there may other considerations first.

**Comment by sunt05** [comment]
This is moved here from SUEWS issues as a more general discussion fits better to this thread.

**Comment by sunt05** [comment]
What specific considerations/interests do you have for dusty days? radiation? deposition of dust? or something else?

**Comment by khorsans** [comment]
Dear Sun,
I am interested in radiation.

**Reply by suegrimmond** [reply]
SUEWS - does not have a detailed radiation model to deal with the radiative changes coming from dust - it would use the output of such a model (or observations)
